### PR TITLE
Cleaner logging page

### PR DESCRIPTION
**Pull request by ericmatte**
### Description

This PR cleans up a little the logging page.
I've also added a <kbd>Download Logs</kbd> button.

### Screenshots

With "A Message from the Developer":

Screen Shot 2020-08-30 at 10 41 53

With the message dismissed:

Screen Shot 2020-08-30 at 10 42 59

### CODE REVIEW

File: hassio-google-drive-backup/backup/static/logs.html [hunk 1]
```diff
@@ -32,18 +32,10 @@
     <script type="text/javascript">
         toasted = false;
         running = false;
-        function resize() {
-            var offset = 0;
-            if (!$('#hello_card').data("closed")) {
-                offset = $("#hello_card").height();
-            }
-            $("#log_container").css("height", ($(window).height() - offset - 50) + "px");
-        }
```
Comment (ericmatte) [inline hassio-google-drive-backup/backup/static/logs.html:41]: By using `style="height: 100%; display: flex; flex-direction: column"` on the wrapper div, we don't need to calculate the height with javascript on resize; CSS does it for us :)

Comment (sabeechen) [reply hassio-google-drive-backup/backup/static/logs.html:41]: Thats great.  You'll probably notice this as you run around the codebase, but my html/javascript is pretty weak.
I'm more of a "backend" kind of programmer, and all my js/css/etc is basically just hacked together until it works :)

**Comment by sabeechen** [comment]
BTW you can ignore the error messages from Travis CI, its angry about a change I made months ago and I've never properly sorted it out.

File: hassio-google-drive-backup/backup/static/logs.html (hunk 1 above)
Comment (ericmatte) [reply hassio-google-drive-backup/backup/static/logs.html:41]: This is not a problem at all :) If I happen to find other hacks that could be cleaned up, I will sure check them out.

**Pull request merged**